.github/workflows/pr_notifier.yml

```diff
@@ -0,0 +1,22 @@
+on:
```

Comment:
I would recommend leaving the manual trigger option that I had in the example. This will allow you to manually trigger it from the UI to test it and also if for whatever reason the run fails and we want to run it.